### ISSUE TITLE
Feature Request: Notifications

### ISSUE THREAD

**Issue by keith8496** [flexdash/node-red-flexdash#15]
Can the features of the "notification" node in [node-red-dashboard](https://github.com/node-red/node-red-dashboard) be implemented in node-red-flexdash?

I would like to send error and exception messages to a pop-up without regard to which tab is open. I noticed an orange pop-up is triggered at the bottom-center of the dashboard whenever the flow is restarted. I basically want to expose that feature so that I can send messages to it.

If I understand the existing popup button correctly, I would have to create a copy of it for each tab for this use-case.

Thanks!

**Comment by tve** [comment]
You are correct, currently the only solution is a pop-up on each tab, which isn't great.
